common-messaging/src/main/scala/org/genivi/sota/messaging/Messages.scala

```diff
@@ -112,6 +112,11 @@ object Messages {
                                     pkgSize: Long, pkgChecksum: String,
                                     timestamp: Instant = Instant.now()) extends BusMessage
 
+  final case class DeltaGenerationRequest(id: Uuid, fromVersion: String, toVersion: String) extends BusMessage
+
+  final case class DeltaGenerated(campaignId: Uuid, fromVersion: String, toVersion: String, size: Long,
+                                  succeeded: Boolean) extends BusMessage
```

Comment:
Hmm @simao it seems that @CalumMcCall expects a `UUID` in the field called `campaignId`, can you change the event in your PR?